web_generate_assets/hooks.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+import os
+
+from odoo import fields, registry
+
+logger = logging.getLogger(__name__)
+
+
+def active_cron_assets():
+    """Plan the next execution of the cron responsible to generate assets."""
+    if os.environ.get("RUNNING_ENV") == "dev":
+        return
+    dbname = os.environ.get("DB_NAME")
+    reg = registry(dbname)
+    with reg.cursor() as cr:
+        cron_module, cron_ref = "web_generate_assets", "cron_generate_assets"
+        query = """
+           SELECT model, res_id
+           FROM ir_model_data
+           WHERE module=%s
+           AND name=%s;
+        """
```

Comment:
Since we could have dedicated services for odoo web/odoo cronjobs, we should a a lock.
Otherwise we may get a "serialize access" error. 
```suggestion
        query = """
           SELECT model, res_id
           FROM ir_model_data
           WHERE module=%s
           AND name=%s
           FOR UPDATE;
        """
```

Reply:
I don't think we should use a "FOR UPDATE" because it locks the complete ir_model_data table. :-)